### ISSUE TITLE
Color intended rules

### ISSUE THREAD

**Issue by Tobi-La** [kuzyn/chatscript-tmlanguage#52]
Since I have more than one topic in every topic file, I would like to intend every rule by 4 spaces to have a better overview where one topic ends and another one begins.

Currently I am not doing this because the syntax highlighting doesn't work for intended rules.

I would be very happy if this would be possible :)

![intended rules](https://cloud.githubusercontent.com/assets/25849067/23199369/3ff0b164-f8ce-11e6-97a8-8ba8212efd2f.PNG)

Thank you :)

**Comment by kuzyn** [comment]
Yeah that's because rejoinders rely on line beginning... Will look into it
